Tasks/Xcode/postxcode.ts

```diff
@@ -9,6 +9,42 @@ async function run() {
     try {
         tl.setResourcePath(path.join(__dirname, 'task.json'));
 
+        //--------------------------------------------------------
+        // Test publishing - publish even if tests fail
+        //--------------------------------------------------------
+        let testResultsFiles: string;
+        const publishResults: boolean = tl.getBoolInput('publishJUnitResults', false);
+        const useXcpretty: boolean = tl.getBoolInput('useXcpretty', false);
+        const workingDir: string = tl.getPathInput('cwd');
+        
+        if (publishResults && useXcpretty) {
+            if (!useXcpretty) {
+                tl.warning(tl.loc('UseXcprettyForTestPublishing'));
+            } else {
+                testResultsFiles = tl.resolve(workingDir, '**/build/reports/junit.xml');
```

Comment:
Given that testResultsFiles is declared as a string, I wonder how multiple matches are handled.

Reply:
If I get a chance I'll step through this code to understand where the wildcards get expanded.